src/scripts/visualizations/tooltip.js

```diff
@@ -97,10 +98,21 @@
                     };
 
                     return clampPosition(pos);
+                },
+
+                'cursor': function cursorPositioner() {
+                    var pos = d3.mouse(layer[0][0]);
+                    var tooltip = tooltipElement[0][0]
+                    var offsetX = 40;
```

Comment:
Looking at this code, I think the same thing: what is this magic number? The only way a developer could find the answer is by finding and reading this PR. It'd be better to make it self-documenting, and might as well leave a comment at the same time:

```js
// magic offset number that position the tooltip nicely on the
// horizontal position regardless of the size of the tooltip
var HORIZONTAL_TOOLTIP_OFFSET = 40;
var offsetX = HORIZONTAL_TOOLTIP_OFFSET;
```